tests/p2p/config.yaml

```diff
@@ -48,7 +48,27 @@ client:
     layer-duration-sec: '60'
     layer-average-size: '20'
     hare-wakeup-delta: '10'
-  deployment_type: deployment # default:deployment 
+  deployment_type: deployment # default:deployment
+client_v2:
+  image: 'spacemeshos/go-spacemesh:go_auto_client_v2'
```

Comment:
we have to find a way to make this test keep running, using this image in the future will be a problem as the code won't nessecarly be compatible

Reply:
do you want me to fix it on this PR?